ephios/core/templates/core/home.html

```diff
@@ -51,9 +79,24 @@ <h5 class="card-title mb-0">{% trans "Events with requested participations" %}</
                 <ul class="list-group list-group-flush">
                     {% for shift in shifts %}
                         <li class="list-group-item flex-column">
-                            <div class="d-flex w-100 justify-content-between align-items-center">
-                                <span class="">{{ shift.event.title }} ({{ shift.get_datetime_display }})</span>
-                                <div>
+                            <div class="row">
+                                <div class="col-auto px-0">
+                                    <span class="text-info fa fa-hand-point-right ps-2"></span>
```

Comment:
![grafik](https://user-images.githubusercontent.com/12702135/177428276-a086f010-6828-4208-af2c-6329d9c74a68.png)
to me it looks like the icon is not perfectly centered?

Reply:
Which direction? Vertical or horizontal?

Reply:
I think it is right, the icon's finger is just not centered